pkgs/applications/audio/tauon/default.nix

```
@@ -23,13 +23,14 @@
 
 stdenv.mkDerivation rec {
   pname = "tauon";
-  version = "7.3.1";
+  version = "7.4.0";
 
   src = fetchFromGitHub {
     owner = "Taiko2k";
     repo = "TauonMusicBox";
     rev = "v${version}";
-    sha256 = "sha256-g3mRVPOXU3N+MApLaHAAIIsVuVv2GeB1Nj//8tuS0oI=";
+    sha256 = "sha256-pat6vkxeAOsh2OmnMreBAhG3AtvrXCCusCO/N1Z4Iy4=";
+    fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
    sha256 = "sha256-pat6vkxeAOsh2OmnMreBAhG3AtvrXCCusCO/N1Z4Iy4=";
```
since it influences the hash but more important we should use our, not vendored versions of those libraries. I just did a quick check and we should have them in nixpkgs already.

---

The two libraries are miniaudio and kissfft, a quick search only yields the python bindings for the former (which presumably vendor the C code) and the latter only provides a dynamic library. And changing either one will require us to completely replace the build script with our own, which I'd like to avoid.

---

> And changing either one will require us to completely replace the build script with our own, which I'd like to avoid.

We can symlink the source files into where tauon expects them. Then we don't need to fiddle with the custom build script and still don't use vendored dependencies.